src/components/molecules/LoginComponent/LoginSection.module.scss

```diff
@@ -0,0 +1,99 @@
+@import "shared";
```

Comment:
Could you review our previous components' scss files? We use our variables. For example, we don't use colors directly, we declare colors in _colors.scss then we use it. It allows us to avoid side effects and easy maintainable